src/dawn/CodeGen/GridTools/ASTCodeGenGTStencilBody.h

```diff
@@ -26,26 +26,29 @@ namespace dawn {
 class StencilInstantiation;
 class StencilFunctionInstantiation;
 
+namespace codegen {
+namespace gt {
+
 /// @brief ASTVisitor to generate C++ gridtools code for the stencil and stencil function bodies
-/// @ingroup codegen
-class ASTCodeGenGTClangStencilBody : public ASTCodeGenCXX {
+/// @ingroup gt
+class ASTCodeGenGTStencilBody : public ASTCodeGenCXX {
```

Comment:
I just removed the CodeGenGT since it is already present in folder and namespace. 
This is the visitor that traverses the Stencil Body AST, so actually ASTStencilBodyVisitor, I would find it more clear

Reply:
that sounds very good. I also think the folder-structure and the namespaces should not be in the class-names. Looks really good now